cpp/src/slice2js/Gen.cpp

```diff
@@ -22,6 +22,38 @@ using namespace IceInternal;
 
 namespace
 {
+    CommentPtr parseComment(const ContainedPtr& p)
+    {
+        // JavaScript TypeDoc doc processor doesn't accept # at the beginning of a link
+        // so we need to remove it.
```

Comment:
Do these generate valid links still?

Reply:
Yes, with Slice and Java doc you can use `{@link #foo} to refer to a member of the class. With TypeDoc is just `{@link foo}`

This was causing warnings mentioned above when parasing https://github.com/zeroc-ice/ice/blob/5d4f24b67aed1cb70412c54dd03035f4179582c4/slice/Ice/Metrics.ice#L134-L136